### PR TITLE
ORG-44: Update readme to mention the blog

### PR DESCRIPTION
**Pull request by reosarevok**
### Implement ORG-44

Since we announce breaking changes in our blog, it makes sense to specify this in our readme.

### CODE REVIEW

**Review by mwiencek — APPROVED** [review]

**Comment by mwiencek** [comment]
We should probably also create a category or tag (like "breaking change") in case people only want to follow those changes specifically (especially if we start posting more stuff to the blog and crowd those out).

File: README.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ For more information, [visit our website](https://musicbrainz.org/doc/About).
 To get help, please join #metabrainz on irc.freenode.net
 
 Important announcements (e.g. about breaking changes to the schema or our 
-API / web service) will be made on [our blog](https://blog.metabrainz.org/),
+API / web service) will be made on
+[our blog](https://blog.metabrainz.org/category/development/updates/breaking-changes/),
```
Comment (yvanzo) [inline README.md:15]: Link more specifically to MusicBrainz breaking changes:

```suggestion
[our blog](https://blog.metabrainz.org/category/musicbrainz+breaking-changes/),
```

I guess that [2020-09-21](https://blog.metabrainz.org/2020/09/21/musicbrainz-server-update-2020-09-21/) should be tag with breaking changes then?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ For more information, [visit our website](https://musicbrainz.org/doc/About).
 
 To get help, please join #metabrainz on irc.freenode.net
 
+Important announcements (e.g. about breaking changes to the schema or our 
+API / web service) will be made on [our blog](https://blog.metabrainz.org/),
```
Comment (yvanzo) [inline README.md]: - +database _schema_
- -parenthesis
- replace _important announcements_ with _breaking changes_ as the link specifically targets that in next commit.

```suggestion
Breaking changes to the database schema or our API / web service will be announced on [our blog](https://blog.metabrainz.org/),
```

**Review by yvanzo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (reosarevok) [reply README.md:15]: I don't think that link works as intended? I'm seeing a lot of stuff not tagged with breaking changes.

Comment (yvanzo) [reply README.md:15]: It works as intended: it shows all previous posts about musicbrainz schema changes as I moved “Schema Change Release” category under “Breaking Changes”.

Comment (reosarevok) [reply README.md:15]: Ooooooh. Magic. Ok then!

**Review by yvanzo — APPROVED** [review]

**Comment by yvanzo** [comment]
Added [2020-09-21's update announcing a breaking change in JSON results of MusicBrainz API](https://blog.metabrainz.org/2020/09/21/musicbrainz-server-update-2020-09-21/) to the “Breaking Changes” category.

**Pull request merged**